src/redteam/commands/discover.ts

```diff
@@ -243,7 +243,12 @@ export async function doTargetPurposeDiscovery(
           `${LOG_PREFIX} Received response from target: ${JSON.stringify(targetResponse, null, 2)}`,
         );
 
-        state.answers.push(targetResponse.output);
+        const response =
+          typeof targetResponse.output === 'object'
+            ? JSON.stringify(targetResponse.output)
+            : targetResponse.output;
+
+        state.answers.push(response);
```

Comment:
can we add type annotations to either `state.answers`, or `targetResponse.output` so typescript would force us to handle the serialization in the future?

Reply:
`state.answers` is `string[]` but we can't confidently define `targetResponse.output` because it's the API response.

Reply:
It would probably look like `Omit<any, Callable` at that point